### PR TITLE
Fix clippy issues

### PR DESCRIPTION
**Pull request by alex-semenyuk**

### CODE REVIEW

**Comment by alex-semenyuk** [comment]
Additional issue with codecov https://app.codecov.io/gh/objectionary/sodg

File: src/lib.rs [hunk 1]
```diff
@@ -18,7 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-#![deny(warnings)]
+#![deny(clippy::needless_borrow)]
```
Comment (yegor256) [inline src/lib.rs:21]: @alex-semenyuk why removing `#![deny(warnings)]`?

Comment (alex-semenyuk) [reply src/lib.rs:21]: Saw this 
note: the lint level is defined here
  --> src/lib.rs:21:9
   |
21 | #![deny(warnings)]
   |         ^^^^^^^^
   = note: `#[deny(clippy::needless_borrow)]` implied by `#[deny(warnings)]`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrow

https://github.com/objectionary/sodg/actions/runs/3233873480/jobs/5298203348#step:7:49

**Comment by yegor256** [comment]
@alex-semenyuk clippy problems fixed already

**Pull request closed without merging**